website/docs/migration-guides/yew/from-0_19_0-to-0_20_0.mdx

```diff
@@ -0,0 +1,8 @@
+---
+title: "From 0.19.0 to 0.20.0"
+---
+
+## `_as_body` variant of `start_app` is removed
+
+This method of controlling body has caused issues in event registration and
+SSR hydration. They have been removed.
```

Comment:
```suggestion
This method of controlling body has caused issues in event registration and
SSR hydration. They have been removed. Read more in the [github issue](https://github.com/yewstack/yew/pull/2346).
```

I feel this explanation is lacking, and one you have in your PR description is way more detailed. So lets just add a link to it.